course/lesson-22-strings/lesson.tres

```diff
@@ -0,0 +1,198 @@
+[gd_resource type="Resource" load_steps=21 format=2]
+
+[ext_resource path="res://resources/Lesson.gd" type="Script" id=1]
+[ext_resource path="res://resources/QuizChoice.gd" type="Script" id=2]
+[ext_resource path="res://resources/ContentBlock.gd" type="Script" id=3]
+[ext_resource path="res://course/Documentation.tres" type="Resource" id=4]
+[ext_resource path="res://resources/Practice.gd" type="Script" id=5]
+
+[sub_resource type="Resource" id=1]
+script = ExtResource( 3 )
+content_id = "res://course/lesson-22-strings/content-jotxF5HU.tres"
+title = ""
+type = 0
+text = "Throughout this course, we've mostly stored numbers in variables. But what if we wanted to store a player's name? Using only numbers for names isn't very user-friendly at all, as 70801 here will tell you."
+visual_element_path = ""
+reverse_blocks = false
+has_separator = false
+
+[sub_resource type="Resource" id=2]
+script = ExtResource( 3 )
+content_id = "res://course/lesson-22-strings/content-xZPxY8VU.tres"
+title = ""
+type = 0
+text = ""
+visual_element_path = "visuals/RobotWithNumberName.tscn"
+reverse_blocks = false
+has_separator = false
+
+[sub_resource type="Resource" id=14]
+script = ExtResource( 3 )
+content_id = "res://course/lesson-22-strings/content-kGx0c7DD.tres"
+title = ""
+type = 0
+text = "This is where strings help us."
+visual_element_path = ""
+reverse_blocks = false
+has_separator = false
+
+[sub_resource type="Resource" id=3]
+script = ExtResource( 3 )
+content_id = "res://course/lesson-22-strings/content-DpPwQDwf.tres"
+title = "What are strings"
+type = 0
+text = "A string is a variable type which holds text instead of numbers. When we use strings, we use quotation marks (\"\") to differentiate them from other variable types, or method names for example.
```

Comment:
You shouldn't talk about variable type, at this stage, variables are dynamic. Also, String itself is a _value_ type, you don't need a variable to hold it.